tests/api/test_superadmin_only.py

```diff
@@ -577,13 +577,14 @@ def test_send_motd_no_primary_email(client: flask.testing.FlaskClient) -> None:
     assert not models.Email.query.filter_by(email=email).one_or_none()
     assert not models.User.query.filter_by(username=username).one().primary_email
 
-    # Attempt request
-    with unittest.mock.patch.object(flask_mail.Connection, "send") as mock_mail_send:
+    # Attempt request and catch email
+    with mail.record_messages() as outbox:
         response: werkzeug.test.WrapperTestResponse = client.post(
             tests.DDSEndpoint.MOTD_SEND, headers=token, json={"motd_id": created_motd.id}
         )
         assert response.status_code == http.HTTPStatus.OK
-        assert mock_mail_send.call_count == num_users - 1
+        assert len(outbox) == num_users - 1
+        assert "Important Information: Data Delivery System" in outbox[-1].subject
```

Comment:
Actually could you also add this check so that we know that it fails when it's suppose to? 
```suggestion
        assert "Important Information: Data Delivery System" in outbox[-1].subject
        assert "incorrect subject" not in outbox[-1].subject
```